slackevents/parsers.go

```diff
@@ -158,7 +158,7 @@ func ParseEvent(rawEvent json.RawMessage, opts ...Option) (EventsAPIEvent, error
 	}
 
 	if !cfg.TokenVerified {
-		return EventsAPIEvent{}, errors.New("No")
+		return EventsAPIEvent{}, errors.New("Invalid verification token")
```

Comment:
Haha I guess I am not the only one who thought that should be changed! https://github.com/whithajess/unity/blob/master/hello/main.go#L30

Reply:
So that was causing the weird "No" error yesterday 😂

Reply:
rofl.